### ISSUE TITLE
No images in Calendar Mode

### ISSUE THREAD

**Issue by Yp-pro** [toutjavascript/Fooocus-Log-Viewer#14]
Hello,

Good job! I like your Log Viewer more than the native one. 

But I've got an issue with Calendar Mode:

![image](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/156006482/b6b21afe-a6f3-408d-876f-423631b4746f)

When I click on a date (pink bar) the Viewer doesn't show images and says "No images" but there are many ones in this folder and the native log shows them.

Windows 10 Pro, Chrome. Other browsers - the same situation

**Comment by toutjavascript** [comment]
Hi
Thanks

It seems linked to this issue (with a proposal of fix)
https://github.com/toutjavascript/Fooocus-Log-Viewer/issues/13

Can you confirm ?
I will merge the fix (even if I cannot reproduce because of international date format display)

**Comment by Yp-pro** [comment]
Sure

**Comment by toutjavascript** [comment]
New release is published with fix

**Comment by Yp-pro** [comment]
Sorry, but it doesn't work.

Here's the Calendar Mode:

![image](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/156006482/805215c3-ce43-4e30-9ce0-10ed76e3da0f)

I checked it in the regular mode, the same issue:

![image](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/156006482/4d0d7a70-dd6b-4051-96bd-84231d01a619)

Of course, there're many images in that folder:

![image](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/156006482/8307573d-cf09-45f4-9d90-a3732129ba3f)

And the native log shows it:

![image](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/156006482/a03c1016-1f35-4e17-818e-cb496b372825)

I tryed it in Chrome and Firefox. No difference.

Sure, I deleted the old files and unzipped the new ones (1.5.2). Fooocus was closed and opened again after update, as you have mentioned. No difference.

**Comment by Yp-pro** [comment]
I have a clue. All images that I can see are PNG. I can't see JPG. Is it important?

**Comment by toutjavascript** [comment]
I think .jpg extension could be the reason
Fooocus generates .png
How do you have .jpg ?

**Comment by Yp-pro** [comment]
> I think .jpg extension could be the reason Fooocus generates .png How do you have .jpg ?

Converted them with IrfanView for saving the space on the disc (I generate a lot)

**Comment by toutjavascript** [comment]
And you change the name in log.html at the same time ?

**Comment by Yp-pro** [comment]
Not name, just an extension

**Comment by toutjavascript** [comment]
yes, you're right, but is the extension changed every where in log.html (not only where it is viewable in browser)

**Comment by Yp-pro** [comment]
I just do Find/Replace and change all "png" to "jpg" in log.html. Everything works well after that (except your Viewer)

**Comment by toutjavascript** [comment]
Log.html is not used by Fooocus itself.
I will check if I am looking for .png file or the exact full name that is stored in log.html

thanx

**Comment by toutjavascript** [comment]
So I checked the log viewer. It works on my computer if I :
1- convert 2024-02-17 files from .png to .jpg with irfanview
2- replace all .png to .jpg in 2024-02-17/log.html file (3 times per image)

They are displayed...
![Capture d’écran 2024-02-17 à 19 08 18](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/30899600/1034ae02-241e-40fd-b530-ec9887b474c4)

We are not lucky with your config :(

**Comment by Yp-pro** [comment]
It seems like I found the problem. Look:

1) When I generate as usual (PNG), everything is okay:

![image](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/156006482/da6b42ee-d0ed-43f3-8e6c-49841514fcad)

2) If I convert these images to JPG (without changing log.html) I see the following:

![image](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/156006482/6961c81b-6345-4078-a1bb-a9ce27e6ffce)

and see nothing if I open log.html in a browser:

![image](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/156006482/2b25a468-1d44-4c09-a02e-8e54fe24acb6)

3) Important! If I do Find/Replace in log.html and replace every "png" there, your Viewer won't show these images.
4) Important! If I rename only image extensions, youe Viewer works well:

![image](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/156006482/af0e69ed-2e8c-43d9-bfa6-31ad659c43ec)

Wrap up:

It's very inportant not to change the div id in this part, only the image extensions:

<div id="**2024-02-17_20-30-34_5910_png**" class="image-container"><hr><table><tr>
<td><a href="2024-02-17_20-30-34_5910.png" target="_blank"><img src='2024-02-17_20-30-34_5910.png' onerror="this.closest('.image-container').style.display='none';" loading='lazy'></img></a><div>2024-02-17_20-30-34_5910.png

It brokes evertything.

* It would be great if you fixed that somehow

**Comment by toutjavascript** [comment]
Well done buddy !

I will try to fix the div#id content to be ok with _png or _jpg
